Makefile

```diff
@@ -12,44 +12,44 @@
 
 IMAGE_NAME := fluent/fluentd-kubernetes
 X86_IMAGES := \
-	v1.11/debian-azureblob:v1.11.4-debian-azureblob-1.0,v1.11-debian-azureblob-1 \
-	v1.11/debian-elasticsearch7:v1.11.4-debian-elasticsearch7-1.0,v1.11-debian-elasticsearch7-1,v1-debian-elasticsearch \
-	v1.11/debian-elasticsearch6:v1.11.4-debian-elasticsearch6-1.0,v1.11-debian-elasticsearch6-1 \
-	v1.11/debian-loggly:v1.11.4-debian-loggly-1.0,v1.11-debian-loggly-1 \
-	v1.11/debian-logentries:v1.11.4-debian-logentries-1.0,v1.11-debian-logentries-1 \
-	v1.11/debian-cloudwatch:v1.11.4-debian-cloudwatch-1.1,v1.11-debian-cloudwatch-1 \
-	v1.11/debian-stackdriver:v1.11.4-debian-stackdriver-1.0,v1.11-debian-stackdriver-1 \
-	v1.11/debian-s3:v1.11.4-debian-s3-1.2,v1.11-debian-s3-1 \
-	v1.11/debian-syslog:v1.11.4-debian-syslog-1.0,v1.11-debian-syslog-1 \
-	v1.11/debian-forward:v1.11.4-debian-forward-1.0,v1.11-debian-forward-1 \
-	v1.11/debian-gcs:v1.11.4-debian-gcs-1.0,v1.11-debian-gcs-1 \
-	v1.11/debian-graylog:v1.11.4-debian-graylog-1.0,v1.11-debian-graylog-1 \
-	v1.11/debian-papertrail:v1.11.4-debian-papertrail-1.0,v1.11-debian-papertrail-1 \
-	v1.11/debian-logzio:v1.11.4-debian-logzio-1.0,v1.11-debian-logzio-1 \
-	v1.11/debian-kafka:v1.11.4-debian-kafka-1.0,v1.11-debian-kafka-1 \
-	v1.11/debian-kafka2:v1.11.4-debian-kafka2-1.0,v1.11-debian-kafka2-1 \
-	v1.11/debian-kinesis:v1.11.4-debian-kinesis-1.0,v1.11-debian-kinesis-1
+	v1.11/debian-azureblob:v1.11.5-debian-azureblob-1.0,v1.11-debian-azureblob-1 \
+	v1.11/debian-elasticsearch7:v1.11.5-debian-elasticsearch7-1.0,v1.11-debian-elasticsearch7-1,v1-debian-elasticsearch \
+	v1.11/debian-elasticsearch6:v1.11.5-debian-elasticsearch6-1.0,v1.11-debian-elasticsearch6-1 \
+	v1.11/debian-loggly:v1.11.5-debian-loggly-1.0,v1.11-debian-loggly-1 \
+	v1.11/debian-logentries:v1.11.5-debian-logentries-1.0,v1.11-debian-logentries-1 \
+	v1.11/debian-cloudwatch:v1.11.5-debian-cloudwatch-1.1,v1.11-debian-cloudwatch-1 \
```

Comment:
Could you use `v1.11.5-debian-cloudwatch-1.0` instead of `v1.11.5-debian-cloudwatch-1.1`?
We use image patch version to handle updating dependencies, base images and so on.